src/index.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable import/prefer-default-export */
+import * as myDom from './dom';
+import * as convert from './convert';
+import * as utilis from './utility';
+import './styles.css';
+
+function UserInput(topic) {
+  this.topic = topic;
+}
+
+myDom.userLocationForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  // const userData = myDom.userLocation.value;
+  // const listValue = new UserInput(userData);
+  // myDom.totalList.push(listValue.topic);
```

Comment:
- [ ] [REQUIRED]  Inline comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history.